proposals/0441-formalize-language-mode-terminology.md

```diff
@@ -3,15 +3,15 @@
 * Proposal: [SE-0441](0441-formalize-language-mode-terminology.md)
 * Author: [James Dempsey](https://github.com/dempseyatgithub)
 * Review Manager: [John McCall](https://github.com/rjmccall)
-* Status: **Active review (July 15th...29th, 2024)**
+* Status: **Accepted** 
 * Implementation: [apple/swift-package-manager#7620](https://github.com/apple/swift-package-manager/pull/7620)
```

Comment:
```suggestion
* Implementation: [swiftlang/swift-package-manager#7620](https://github.com/swiftlang/swift-package-manager/pull/7620), [swiftlang/swift#75564](https://github.com/swiftlang/swift/pull/75564)
```

---

If you create a new [feedback](https://forums.swift.org/t/se-0441-formalize-language-mode-terminology/73182/13), Xcode Playground Settings (in the File inspector) also has a Swift Version picker.

Reply:
Change made in 540b9c6c77a48fefd68ca248daba749b6ed22875